proprietary/Components/src/denhaag/typography.tokens.json

```diff
@@ -0,0 +1,36 @@
+{
+  "denhaag": {
+    "typography": {
```

Comment:
Could you explain the decision to add these tokens? For the existing typography implementation, we've made a very conscious choice of using Utrecht tokens to validate their usability and make it easier to move to a NL Design System maintained typography component in the future. What options were you missing in the original implementation?

Reply:
@bddjong 
I thought the design tokens where supposed to be added by default in: proprietary/Components/src/denhaag/.
But the values to be recycled if they already exist in denhaag.

I'll check again, to remove unnecessary tokens.